src/main/java/domain/Card.java

```diff
@@ -0,0 +1,40 @@
+package domain;
+
+import java.util.Objects;
+
+public class Card {
+
+    private final Symbol symbol;
+    private final Number number;
+
+    public Card(Symbol symbol, Number number) {
+        this.symbol = symbol;
+        this.number = number;
+    }
+
+    public boolean isAce() {
+        return number.equals(Number.ACE);
```

Comment:
enum class의 경우 동일성 판단을해도 동일한 결과를 얻을 수 있어요. 
```suggestion
        return number == Number.ACE;
```

Reply:
`equals`보다는 메모리 주소로 비교하는 `==`이 미세하지만 성능적인 이점이 존재하며 null의 경우도 안전하게 체크 가능하다는 것을 알게 되었습니다! 감사합니다